lib/bolt/transport/powershell.rb

```diff
@@ -57,11 +57,16 @@ def execute_process(path, arguments, stdin = nil)
             "'" + arg.gsub("'", "''") + "'"
           end.join(' ')
 
+          quoted_path = if path =~ /^'.*'$/ || path =~ /^".*"$/
+                          path
+                        else
+                          "\"#{path}\""
```

Comment:
Should we copy the logic from above that we use for arguments?

```rb
"'" + arg.gsub("'", "''") + "'"
```

Reply:
I tried that, it does not work. The literal string does not execute the task.

Reply:
```
PS C:\program files\Miniconda3> '"C:/Program Files/Miniconda3/python.exe"'
"C:/Program Files/Miniconda3/python.exe"
```

Reply:
You'd need to use the `&` command to force it to execute.  Whether this should be used in this case is debatable

```
C:\Source> & '"C:/Program Files/Miniconda3/python.exe"'
& : The term '"C:/Program Files/Miniconda3/python.exe"' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if
a path was included, verify that the path is correct and try again.
At line:1 char:3
+ & '"C:/Program Files/Miniconda3/python.exe"'
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: ("C:/Program Files/Miniconda3/python.exe":String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

C:\Source> '"C:/Program Files/Miniconda3/python.exe"'
"C:/Program Files/Miniconda3/python.exe"
```